go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/sigstore/fulcio
 
-go 1.22.6
+go 1.23
```

Comment:
Is this fine without the specific version or toolchain? I would prefer this but not sure if go.mod requires otherwise.